CancerGov/grunt/sprite.js

```diff
@@ -12,28 +12,28 @@ module.exports = function (grunt, options) {
             imgPath: dirs.dist.sprites
 
         },
-        carousel: {
-            imgPath: imgPath + 'carousel-bgs.jpg',
-            src: [dirs.src.sprites + 'carousel-bgs/*.{png,jpg,gif}' ],
-            dest: dirs.dist.sprites + 'carousel-bgs.jpg',
-            destCss: dirs.src.modules + 'carousel/_carousel-sprite-map.scss',
-            algorithm: 'top-down',
-            cssOpts: {functions: false},
-            imgOpts: {quality: 75},
-            cssSpritesheetName: 'carousel'
-        },
-        accordion: {
-            imgPath: imgPath + 'accordion-bgs.jpg',
-            src: [dirs.src.sprites + 'accordion-bgs/*.{png,jpg,gif}' ],
-            dest: dirs.dist.sprites + 'accordion-bgs.jpg',
-            destCss: dirs.src.modules + 'accordion/_accordion-sprite-map.scss',
-            algorithm: 'top-down',
-            cssOpts: {functions: false},
-            imgOpts: {quality: 75},
-            cssSpritesheetName: 'accordion',
-            cssVarMap: function (sprite) {
-                sprite.name = 'accordion-' + sprite.name;
-            }
-        }
+        // carousel: {
```

Comment:
These tasks do not need to be commented out. It's enough that the tasks were removed from the Gruntfile.js. If we're going to remove the task completely then we should probably remove the image folders they src as well.

Reply:
Yeah, but I just wanted to guilt the next guy down the line into deleting them.